packages/react-router/__tests__/router/browser-test.ts

```diff
@@ -2,8 +2,10 @@
 
 import { JSDOM } from "jsdom";
 
-import type { BrowserHistory } from "../../lib/router";
-import { createBrowserHistory } from "../../lib/router";
+import {
+  type BrowserHistory,
+  createBrowserHistory,
+} from "../../lib/router/history";
```

Comment:
All router imports come from the source files, no more `lib/router/index.ts` barrel file to re-export